frontend/tests/unit/composables/useToast.test.ts

```
@@ -0,0 +1,345 @@
+import { useToast } from '@/core/composables/useToast'
+
+describe('useToast', () => {
+  beforeEach(() => {
+    vi.useFakeTimers()
+  })
+
+  afterEach(() => {
+    vi.runOnlyPendingTimers()
+    vi.useRealTimers()
+    const { toasts, remove } = useToast()
+    toasts.value.forEach((t) => remove(t.id))
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)